python/py-codecov/Portfile

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        codecov codecov-python 2.1.11 v
+name                py-codecov
+revision            0
+categories-append   devel
+
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
+supported_archs     noarch
+
+description         Hosted coverage reports for Github, Bitbucket and Gitlab
+
+long_description    ${description}.
+
+checksums           rmd160  4075f26b77ad55651a8a560efdeea850dd523700 \
+                    sha256  5bae4cd0965d8d7e2c94552a9e98632ab1c36feec48b792d740ba0985ba750a8 \
+                    size    26437
+
+python.versions     37 38
```

Comment:
Steve, do all of these modules support Python 3.9? If so, can you add that support for each? (Or at least, for the ones that definitively support it.)

Reply:
Pretty sure. Let's do the version update first and I'll add the Python bindings in a separate PR.